### PR TITLE
OM_PRAC-18 Tutor search page

### PR DESCRIPTION
**Pull request by oxangyal**
## One Line Description
Create a search page where user can see all tutors and be able to input teaching subjects filter available tutors.
## Requirements
Responsive layout of the tutor search page displaying all registered tutors
The page should have an input field where all users can enter teaching subjects
Upon entering the teaching subjects and submitting the form, a list of tutors who teach those subjects should be displayed.
Each Tutor card  contains information about tutor
_What this code should do_

## Notes

_Any additional info_

## Test Steps
Navigate to http://localhost:5173/tutorsearch.
Enter a teaching subject into the input field.
Submit the form and verify that a list of tutors teaching the entered subject is displayed.
_Instructions on how to test the changes_

## UI/UX Outcome
see Figma

_Attach a screenshot here_
searchpage

### CODE REVIEW

**Review by IrynaKolh — APPROVED** [review]
LGFM

File: src/components/TutorCard.tsx [hunk 1]
```diff
@@ -76,177 +86,172 @@ const TutorCard: React.FC = () => {
...
                             mx="auto"
                             style={customAvatarStyle}
                         />
                     </Stack>
                     <Text mt="4" w="100%" fontSize={theme.dashboardButtons.fontSize}>
+                        {tutor.about}
+                    </Text>
+                    {/* <CardBody> */}
+                    {/* <Stack spacing="2" textAlign="center">
+                        <Heading size="xs">Patrick Hill</Heading>
```
Comment (katsiarynalashcheuskaya) [inline src/components/TutorCard.tsx]: It seems that this commented-out code is no longer needed. Let's remove it to maintain a code clean.

File: src/components/TutorCard.tsx [hunk 2]
```diff
@@ -76,177 +86,172 @@ const TutorCard: React.FC = () => {
...
-                                    m="0"
-                                    w="50%"
-                                    _hover={{ bg: 'red.500' }}
-                                >
+
+                                    {/* <Text>
+                                    {[
+                                        tutor.MathSubject,
+                                        tutor.ForeignLanguages,
+                                        tutor.Science,
```
Comment (katsiarynalashcheuskaya) [inline src/components/TutorCard.tsx]: The same here.

File: src/pages/TutorSearchPage.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                templateColumns={{
+                    base: 'repeat(1, 1fr)',
+                    sm: 'repeat(2, 1fr)',
+                    md: 'repeat(3, 1fr)',
+                }}
+                justifyItems="center"
+            >
+                {/* <Grid
+                gap={10}
+                templateColumns={{ base: 'repeat(1, 1fr)', sm: 'repeat(2, 1fr)', md: 'repeat(3, 1fr)' }}
```
Comment (katsiarynalashcheuskaya) [inline src/pages/TutorSearchPage.tsx:60]: And here too.

**Review by katsiarynalashcheuskaya — APPROVED** [review]
Just a small observation, otherwise it looks great! Well done, Oxana!

File: src/pages/TutorSearchPage.tsx (hunk 3 above)
Comment (oxangyal) [reply src/pages/TutorSearchPage.tsx:60]: I need to leave this one. Comment added.

File: src/components/TutorCard.tsx (hunk 2 above)
Comment (oxangyal) [reply src/components/TutorCard.tsx]: Done

File: src/components/TutorCard.tsx (hunk 1 above)
Comment (oxangyal) [reply src/components/TutorCard.tsx]: Done

**Pull request merged**